homeworks/rostyslav.vyshemirskyi_Rooosss/PokemonProject/scripts/calculator.js

```diff
@@ -0,0 +1,46 @@
+/**
+ *
+ * @param firstValue string with first number
+ * @param secondValue string with second number
+ * @param operation string with operation symbol
+ * @returns {string|pokemons} result of calculation or error message:
```

Comment:
In the `{}` you provide type for [jsdoc returns](https://jsdoc.app/tags-returns)
Now you state, that this function returns `string` or `pokemon` type, which is incorrect.
Please remove `pokemon`

Reply:
@Rooosss, please correct in the next task